submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from helpers import FileManager, Vocabulary, EpithetGenerator
+
+test_path = ('/Users/ethansternke/Documents/Kenzie/back-end'
+             '/backend-epithet-generator/resources/test.json')
+
+test_vocab = (
+    {
+        'Column 2': ['word2'],
+        'Column 3': ['word3'],
+        'Column 1': ['word1']
+    },
+    [
+        'Column 2',
+        'Column 3',
+        'Column 1'
+    ]
+)
+
+
+class TestFileManager(unittest.TestCase):
+    def test_get_extension(self):
+        self.assertEqual(FileManager.get_extension(test_path), 'json')
+
+    def test_read_json(self):
+        self.assertEqual(FileManager.read_json(test_path), {
+            'Column 2': ['word2'],
+            'Column 3': ['word3'],
+            'Column 1': ['word1']
+        })
+
+
+class TestVocabulary(unittest.TestCase):
+    def test_from_file(self):
+        self.assertEqual(Vocabulary.from_file(test_path), test_vocab)
+
+    def test_from_json(self):
+        self.assertEqual(Vocabulary.from_json(test_path), test_vocab)
+
+    def test_strategies(self):
+        self.assertTrue(Vocabulary.strategies('json'))
```

Comment:
What does this test? Calling strategies alone returns a strategy function. In this case, it returns the class method from_json. To test strategies you'd need to

```python
self.assertTrue(Vocabulary.strategies('json'), Vocabulary.from_json)
```